thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/views/AnomalyResultView.js

```diff
@@ -137,8 +137,8 @@ AnomalyResultView.prototype = {
     // TIME RANGE SELECTION
     this.timeRangeConfig.startDate = this.anomalyResultModel.startDate;
     this.timeRangeConfig.endDate = this.anomalyResultModel.endDate;
-    function cb(start, end) {
-      $('#anomalies-time-range span').addClass("time-range").html(start.format('MMM D, ') + start.format('hh a') + '  &mdash;  ' + end.format('MMM D, ') + end.format('hh a'));
+    function cb(start, end, rangeType = constants.DATE_RANGE_CUSTOM) {
+      $('#anomalies-time-range span').addClass("time-range").html(`<span>${rangeType}</span> ${start.format(constants.DATE_RANGE_FORMAT)} &mdash; ${end.format(constants.DATE_RANGE_FORMAT)}`);
```

Comment:
same here. I prefer to have these jquery selectors at initializing and reassign once it's rerendered